### PR TITLE
added make command to run pytest nbval

### LINKED ISSUE #49: Test notebooks in Travis-CI
## Description

Similar to what is done in Raven.

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #49

Changes:

* Added make command to test notebooks.

### CODE REVIEW

**Comment by huard** [comment]
@Zeitsperre Could you look into this build failure ?

**Comment by Zeitsperre** [comment]
@huard On it.

**Comment by davidcaron** [comment]
You need `libnetcdf==4.6.2` like on the master branch.

**Comment by Zeitsperre** [comment]
@davidcaron master branch of this PR or of xclim conda recipe?

**Comment by huard** [comment]
So we need to include it or it's going to get installed with xarray anyway?

**Comment by davidcaron** [comment]
This one: 

https://github.com/bird-house/finch/blob/3e2d65b3bbe54d032ced85defc9ed2bb14c71fe1/environment.yml#L14

This is a bug in libnetcdf 4.7.x, related to https://github.com/bird-house/finch/issues/54

Somehow, when you merged the master branch, this line wasn't merged.

File: Makefile [hunk 1]
```diff
@@ -108,6 +109,11 @@ test: check_conda
 	@echo "Running tests (skip slow and online tests) ..."
 	@-bash -c "source $(ANACONDA_HOME)/bin/activate $(CONDA_ENV);pytest -v -m 'not slow and not online'"
 
+.PHONY: test_nb
+test_nb:
+	@echo "Running notebook-based tests"
+	@bash -c "source $(ANACONDA_HOME)/bin/activate $(CONDA_ENV);env FINCH_WPS_URL=$(RAVEN_WPS_URL) pytest --nbval $(CURDIR)/docs/source/notebooks/ --sanitize-with $(CURDIR)/docs/source/output_sanitize.cfg --ignore $(CURDIR)/docs/source/notebooks/.ipynb_checkpoints"
```
Comment (davidcaron) [inline Makefile]: `FINCH_WPS_URL=$(RAVEN_WPS_URL)` -> `FINCH_WPS_URL=$(FINCH_WPS_URL)`

Comment (huard) [reply Makefile]: Sorry, this PR is a mess. I thought it was done, but it's a WIP. Will ping you when it's working.

**Comment by huard** [comment]
@Zeitsperre I'm getting conda installation problems here when I try to install locally. Could you take a look ?

**Comment by davidcaron** [comment]
Closed by mistakenly duplicated https://github.com/bird-house/finch/pull/112

**Pull request closed without merging**